pages/opam-using.html

```diff
@@ -16,76 +16,91 @@ <h2>What is opam?</h2>
 provides interactive scripts that allow installing Coq and a standard
 set of packages through opam without having to learn anything about
-  opam.</p>
+opam.</p>
 
-<p>Especially if you are a first-time user, we recommend that you use
+<p>If a standard setup works for you, then we recommend that you use
 these <a href="https://github.com/coq/platform">scripts</a>.  If you
 do, you can skip directly to <a href="#coq-packages">Using opam to
 install Coq packages</a> to learn how to add additional packages to
 the initial package set provided by the platform.</p>
 
 <p>Note that the platform scripts are compatible with existing opam
-installations. They will create a fresh <a href="#switch">switch.</a></p>
+installations. They will create a
+fresh <a href="#switch">switch.</a></p>
 
 <p>If you prefer to do a fully manual installation, you can proceed to
 the next section.</p>
 
 <h2>Initializing opam</h2>
 
-<p>Once opam is installed, it must be initialized before first usage:</p>
+<p>Once opam is installed, it must be initialized before first
+usage:</p>
 
-<pre><code>opam init
-eval $(opam env)
+<pre><code>opam init eval $(opam env)
 </code></pre>
 
 <p><code>opam init</code> will prompt you to allow opam to set up
-initialization scripts, which is generally fine to accept. Otherwise, every
-time a new shell is opened you have to type in the
+initialization scripts, which is generally fine to accept. Otherwise,
+every time a new shell is opened you have to type in the
 <code>eval $(opam env)</code> command above to update environment variables.</p>
 
-<p>By default, opam will use the global installation of OCaml. You can initialize
-opam with an explicit compiler version, for example <#OCAMLV>, with the option
+<p>By default, opam will use the global installation of OCaml. You can
+initialize opam with an explicit compiler version, for example
+<#OCAMLV>, with the option
 <code>--compiler=ocaml-base-compiler.<#OCAMLV></code>.
 See also the section "Managing different versions of OCaml and Coq" below,
 about switches and roots.
 </p>
 
 <h2>Installing Coq</h2>
 
-<p>To install Coq, simply run the following command. Note that installing Coq using opam will build it from sources,
-which will take several minutes to complete:</p>
+<p>To install Coq, simply run the following command. Note that
+installing Coq using opam will build it from sources, which will take
+several minutes to complete:</p>
 
-<pre><code># Pin the coq package to version <#CURRENTVERSION> and install it.
-opam pin add coq <#CURRENTVERSION>
+<pre><code># Pin the coq package to version <#CURRENTVERSION> and
+install it.  opam pin add coq <#CURRENTVERSION>
 </code></pre>
 
-<p>Pinning prevents opam from upgrading Coq automatically, which may cause
-inadvertent breakage in your Coq projects. You can upgrade Coq explicitly to
+<p>Pinning prevents opam from upgrading Coq automatically, which may
+cause inadvertent breakage in your Coq projects. You can upgrade Coq
```

Comment:
I wonder if that wouldn't be clearer like this:
```suggestion
<p>Pinning prevents opam from upgrading Coq automatically, to avoid
causing inadvertent breakage in your Coq projects. You can upgrade Coq
```